proxy/template.go

```diff
@@ -145,9 +145,12 @@ backend {{$.ServiceName}}-be{{.Port}}_{{.Index}}
 		{{- end}}
 		{{- if .DenyHttp}}
     http-request deny if !{ ssl_fc }
-		{{- end}}
-		{{- if .HttpsOnly}}
+        {{- end}}
+        {{- if .HttpsRedirectCode}}
+    redirect scheme https code {{.HttpsRedirectCode}} if !{ ssl_fc }
+		{{- else if .HttpsOnly}}
     redirect scheme https if !{ ssl_fc }
```

Comment:
How about someting like:

```go
{{- if .HttpsRedirectCode}}
    redirect scheme https {{if .HttpsRedirectCode}}code {{.HttpsRedirectCode}}{{end}} if !{ ssl_fc }
{{- end}}
```

That would mean that `HttpsRedirectCode` is used only if `HttpsOnly`is `true`.

What do you think?

Reply:
`if` was wrong. The snippet should be OK now.

Reply:
I agree 👍